library/ix-dev/community/rsyncd/questions.yaml

```diff
@@ -0,0 +1,235 @@
+groups:
+  - name: Rsync Configuration
+    description: Configure Rsync
+  - name: Network Configuration
+    description: Configure Network for Rsync
+  - name: Module Configuration
+    description: Configure Modules for Rsync
+  - name: Resources Configuration
+    description: Configure Resources for Rsync
+
+questions:
+  - variable: rsyncConfig
+    label: ""
+    group: Rsync Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: logToStdout
```

Comment:
Do you see any value to logging this to a file?
And if it is to be logged to some file then we should be doing something custom which exposes a convenient method for a user to know that his logs are being saved at X location

Reply:
Updated the wording, this is mostly need to be togglable in case someone want to use syslog.
(If log file is set, syslog is ignored according to rsyncd docs).

Reply:
I don't follow?
In this case the logs will not be redirected to some file instead?

Reply:
You are correct. Mixed it up with **r**syslog.

In this case, we can either 
1. Remove the toggle for now until we have an actual use case for logging, 
2. Allow the user to add additional volumes (and combine those with the aux parameters). As there are multiple ways to configure logging and some options can't exist at the same time. (eg having log file set, syslog is ignored.)

Reply:
Let's remove it please..we should be making the process easier - so if we were to add it, there should be automatic configuration of volume here etc and i think as we don't have an actual usage atm we should just leave it.

Reply:
Done